virtual-event/package.json

```diff
@@ -11,7 +11,7 @@
     "react-native-animated-emoji": "0.1.4",
     "react-scripts": "3.4.3",
     "stream-chat": "^2.12.0",
-    "stream-chat-react": "3.1.2^",
+    "stream-chat-react": "^3.1.2",
```

Comment:
I believe we had frozen the release since we were about to make the breaking change of removing the immutable dependency in both the component library and JS client. Grant and Anna are about to upgrade the demo apps to TypeScript and the latest versions of each, so at that point I do think it would make sense to set up a recurring update task.

Reply:
that sounds good!